superset-frontend/src/dashboard/components/dnd/DragDroppable.jsx

```diff
@@ -73,6 +74,64 @@ const defaultProps = {
   dragPreviewRef() {},
 };
 
+const DragDroppableStyles = styled.div`
+  ${({ theme }) => css`
+    position: relative;
+
+    &.dragdroppable--dragging {
+      opacity: 0.2;
+    }
+
+    &.dragdroppable-row {
+      width: 100%;
+    }
+
+    &.dragdroppable-column .resizable-container span div {
+      z-index: 10;
```

Comment:
Again, we should have a z-index registry, so we can think of all of Superset as layers in a consolidated way, do calculations like `theme.layers.navigation + 1`.

Reply:
👍